stable/rocketchat/templates/deployment.yaml

```diff
@@ -15,6 +15,9 @@ spec:
         app: {{ template "rocketchat.name" . }}
         release: "{{ .Release.Name }}"
     spec:
+      securityContext:
+        runAsUser: 999
```

Comment:
these are probably best externalized into values.yaml as they could change in future